.github/workflows/test.yml

```diff
@@ -16,4 +16,4 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
       - name: Test with Gradle
-        run: ./gradlew build
+        run: ./gradlew check
```

Comment:
I'm curious why this was `build` previously and what the implications are of this change.

Reply:
See the commit message where this changed: https://github.com/BrightSpots/rcv/pull/587/commits/f9026074f8da97f70fe0692c13b5d17c7ab8353d.